src/shell/components/InstanceFlyoutMenuModal/index.tsx

```diff
@@ -125,6 +137,10 @@ const InstanceFlyoutMenuModal = ({
         padding={2}
         sx={{
           backgroundColor: "grey.100",
+          position: "absolute",
```

Comment:
Currently I didn't put any height on the content that contains the list items. That's why I decided to force this component to be on the bottom. Do you think it's better if I put an initial height for the content body to avoid doing this?

Reply:
Resolved in latest commit.